week2/project/ecommerce/src/pages/productsList/categories/Category.jsx

```diff
@@ -0,0 +1,7 @@
+function Category  ({data, className, onClick})  {
+    return (
+        <div className={`category-item ${className}`} onClick={() => onClick(data)}>{data}</div>
```

Comment:
I think you mean `categoryName` ? because data made me think it's an object of some data !!
📝advise📝

Reply:
Right, thank you
fixed